frontend/src/components/Header.tsx

```diff
@@ -167,52 +199,31 @@ const NavLinks = ({
             <NavLink
               href={childLink.href}
               key={childLink.href}
-              onClick={closeMobileNav}
+              onClick={closeDropdownAndMobileNav}
               text={childLink.text}
             />
           );
         });
         return (
-          <>
-            <NavDropDownButton
-              label={link.text}
-              menuId={link.text}
-              isOpen={secondaryNavOpen}
-              onClick={(e) => {
-                if (!secondaryNavOpen) {
-                  setSecondaryNavOpen(true);
-                  e.stopPropagation();
-                  requestAnimationFrame(() =>
-                    document.addEventListener(
-                      "click",
-                      () => {
-                        setSecondaryNavOpen(false);
-                      },
-                      { once: true },
-                    ),
-                  );
-                }
-              }}
-              onToggle={noop}
-              className={clsx({
-                "usa-current": currentNavItemIndex === index,
-                "simpler-subnav-open": secondaryNavOpen,
-              })}
-            />
-            <Menu
-              id={link.text}
-              items={items}
-              isOpen={secondaryNavOpen}
-              className="margin-top-05"
-            />
-          </>
+          <NavDropdown
+            key={link.href}
+            activeNavDropdownIndex={activeNavDropdownIndex}
+            index={index}
+            isCurrent={currentNavItemIndex === index}
+            linkText={link.text}
+            menuItems={items}
+            setActiveNavDropdownIndex={setActiveNavDropdownIndex}
+          />
         );
       }
       return (
         <NavLink
           href={link.href}
           key={link.href}
-          onClick={closeMobileNav}
+          onClick={() => {
+            closeMobileNav;
```

Comment:
did you mean to call this?

Reply:
No, this was from the merge. It has been reverted back.